clients/src/main/java/org/apache/kafka/clients/admin/AdminClientConfig.java

```diff
@@ -162,7 +168,9 @@
                                 .define(CommonClientConfigs.CLIENT_DNS_LOOKUP_CONFIG,
```

Comment:
Change from `CommonClientConfigs.CLIENT_DNS_LOOKUP_CONFIG` to `CLIENT_DNS_LOOKUP_CONFIG`?